### DISCUSSION TITLE
Errored - Please verify your email address to run GitHub Actions workflows

### DISCUSSION THREAD

**Discussion by manusa** [community/community#26045] [Actions]
<pre><code>### ERRORED 06:57:59Z

- Please verify your email address to run GitHub Actions workflows. https://github.com/settings/emails
</code></pre>
<p>For some reason, our <a href="https://github.com/fabric8io/kubernetes-client/blob/master/.github/workflows/sonar.yml" rel="nofollow noopener">workflow</a>, is failing when run in the <a href="https://github.com/fabric8io/kubernetes-client/runs/471733763" rel="nofollow noopener">main repo</a>, but <a href="https://github.com/manusa/kubernetes-client/actions/runs/45535027" rel="nofollow noopener">not</a> from the <a href="https://github.com/fabric8io/kubernetes-client/pull/2027" rel="nofollow noopener">PR branch</a>.</p>
<p>The error message is the one above, is it possible that’s related to some missing organization settings?</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Could you check your email settings page: <a href="https://github.com/settings/emails?" rel="nofollow noopener">https://github.com/settings/emails? </a></p>
<p>Have you verified your email? </p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/6/6a348df6ef9e936a6619e5a55bdac5d8c75da52b.png" data-download-href="/uploads/short-url/f9xcZcyqu9o6SgEKRitbQw4spET.png?dl=1" title="settings email.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24806478/181094950-0f4d43f1-af0a-4ca4-ba92-79608a9ac3b1.png" alt="settings email.png" data-base62-sha1="f9xcZcyqu9o6SgEKRitbQw4spET" width="690" height="288" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/6/6a348df6ef9e936a6619e5a55bdac5d8c75da52b_2_690x288.png, https://aws1.discourse-cdn.com/github/optimized/2X/6/6a348df6ef9e936a6619e5a55bdac5d8c75da52b_2_1035x432.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/6/6a348df6ef9e936a6619e5a55bdac5d8c75da52b.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/6/6a348df6ef9e936a6619e5a55bdac5d8c75da52b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">settings email.png</span><span class="informations">1098×459 25.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by manusa** [comment]
<p>My personal profile e-mail settings are fine (I run actions on lot’s of repositories e.g.<a href="https://github.com/manusa/electronim/actions" rel="nofollow noopener">https://github.com/manusa/electronim/actions</a>)</p>
<p>I can also run actions on other organization repos (e.g. <a href="https://github.com/marcnuri-demo/actions-remote-dispatch-a/actions" rel="nofollow noopener">https://github.com/marcnuri-demo/actions-remote-dispatch-a/actions</a>)</p>
<p>But for some reason they are still failing for fabric8io/kubernetes-client :frowning:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/d/d550b3c027faa46c2e5e3543eae25d92f8a07379.jpeg" data-download-href="/uploads/short-url/ur4BuIKUBuKlmIhxMBVOdQ4urFf.jpg?dl=1" title="org_data.jpg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/488391/181094953-5550e6f5-d7a0-455d-8f91-22ad359cbefe.jpg" alt="org_data.jpg" data-base62-sha1="ur4BuIKUBuKlmIhxMBVOdQ4urFf" width="490" height="499" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/d/d550b3c027faa46c2e5e3543eae25d92f8a07379_2_490x499.jpg, https://aws1.discourse-cdn.com/github/optimized/2X/d/d550b3c027faa46c2e5e3543eae25d92f8a07379_2_735x748.jpg 1.5x, https://aws1.discourse-cdn.com/github/original/2X/d/d550b3c027faa46c2e5e3543eae25d92f8a07379.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/d550b3c027faa46c2e5e3543eae25d92f8a07379_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">org_data.jpg</span><span class="informations">949×968 78.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by Yanjingzhu** [comment]
<p>Can you run actions in other repositories in fabric8io? Please try to disable actions for fabric8io/kubernetes-client, then enable again, could this help? </p>
<p>If you need to resolve it in hurry, you could contact to Github Support <a href="https://support.github.com/" rel="nofollow noopener">https://support.github.com/</a> . </p>

**Comment by manusa** [comment]
<p>Seems like disabling and enabling GH Actions in the affected repo did the trick. Will confirm when next merge to master finishes.</p>
<p>Thnx!</p>

**Comment by manusa** [comment]
<p>I think we’ve narrowed down the problem.</p>
<p>Probably our <a href="https://github.com/fusesource-ci" rel="nofollow noopener">bot user</a> is the one missing the e-mail address.</p>

**Comment by Yanjingzhu** [comment]
<p>I found that you have resolved your issue. The workflow could be triggered in your repo kubernetes-client now. Congratulations! </p>

**Comment by manusa** [comment]
<p>Workflow actions are finally working.</p>
<p>The missing email configuration was on the user issuing the GitHub Action:</p>
<p><img src="https://user-images.githubusercontent.com/488391/181094960-3958a02c-2b66-42a2-b8cd-e581a3edf484.jpeg" alt="fusesource-ci.jpg" data-base62-sha1="9O7QlqDnEMIVFDaj6EIFWYcteIL" width="158" height="49"></p>
<p>So, yes, modifying <strong>this user’s</strong> email settings worked.</p>
<p>Thanks!</p>

**Comment by gang-gang-gra** [comment]
code anywhere. com wants my phonenuber but I don't have one
